### PR TITLE
feat: add helper util package for overrides

### PR DESCRIPTION
**Pull request by kaustavghosh06**
Note: This PR is opened against the ext branch and not the master branch.

#### Description of changes
Adding  package skeleton for the overrides helper utility

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-cli-overrides-helper/package.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
+{
+    "name": "amplify-cli-overrides-helper",
```
Comment (akshbhu) [inline packages/amplify-cli-overrides-helper/package.json:2]: shoudl we namespace it @aws-amplify/amplify-cli-overrides-helper"?

**Review by akshbhu — APPROVED** [review]
LGTM!!

File: packages/amplify-cli-overrides-helper/package.json (hunk 1 above)
Comment (kaustavghosh06) [reply packages/amplify-cli-overrides-helper/package.json]: Yeah, you're right. Name-spaced it - @aws-amplify/cli-overrides-helper

**Review by sachscode — APPROVED** [review]
LGTM

**Pull request merged**